plugins/uefi-capsule/fu-uefi-common.h

```diff
@@ -9,6 +9,7 @@
 
 #include <fwupdplugin.h>
 
+#include <efivar/efivar-types.h>
```

Comment:
seems you can't include this header in all cases, it should be part of the ifdef?

Reply:
I should have used the main header; fixed.